src/plugins/react.ts

```diff
@@ -6,7 +6,7 @@ import { Plugin, PluginKey } from "prosemirror-state";
  */
 export const ROOT_NODE_KEY = Symbol("@nytimes/react-prosemirror/root-node-key");
 
-export type NodeKey = string | typeof ROOT_NODE_KEY;
+export type NodeKey = string | symbol;
```

Comment:
This was necessary because React doesn't want to let us make an interface type that's indexable by a unique symbol. So our interface is this:

```
export interface NodeViewsContextValue {
  [key: symbol | string]: NodeViewRegistration[];
}
```

I should probably use `NodeKey` in there, eh? I'll push a commit for it.